src/tools/pathTools.cpp

```diff
@@ -302,7 +302,7 @@ std::string kiwix::getFileContent(const std::string& path)
   return content;
 }
 
-bool kiwix::fileExists(const std::string& path)
+bool kiwix::fileExistsAndReadable(const std::string& path)
 {
 #ifdef _WIN32
   return PathFileExistsW(Utf8ToWide(path).c_str());
```

Comment:
On windows, we only check that the file exists. Not it is readable.

Reply:
So, both functions should be kept? `fileExists` and `fileExistsAndReadable` (renamed to `fileReadable` maybe)

Reply:
The function should be fixed.
It doesn't do the same thing on windows and linux.
When it was called `fileExists`, it was also checking the file was readable on linux but we could consider that a implementation details.

But if we rename it to `fileExistsAndReadable` we MUST check that the file is readable (on all platforms).